vue/src/Tag/TagList.vue

```diff
@@ -390,14 +404,31 @@ export default defineComponent({
       return TagsStore.tags.value;
     },
     sortedTags() {
-      const sorted = [...this.tags];
-      sorted.sort((lhs, rhs) => {
+      const searchFilter = this.tagSearch.toLowerCase();
+
+      // look through string properties of custom reports for values that have searchFilter in them
```

Comment:
Not a big deal, but I'm guessing this is a copy/paste error?
```suggestion
      // look through string properties of tags for values that have searchFilter in them
```

Reply:
Thanks done :+1: